project.clj

```diff
@@ -4,32 +4,32 @@
   :license {:name "MIT"
             :url  "https://opensource.org/licenses/MIT"}
 
-  :dependencies [[com.datomic/datomic-free "0.9.5206" :exclusions [joda-time]]
-                 [com.taoensso/timbre "4.3.1"]
+  :dependencies [
+                 [com.taoensso/timbre "4.7.4"]
                  [commons-codec "1.10"]
                  [org.clojure/clojure "1.9.0-alpha14" :scope "provided"]
                  [org.clojure/clojurescript "1.9.473" :scope "provided"]
                  [org.omcljs/om "1.0.0-alpha48" :scope "provided"]
-                 [binaryage/devtools "0.5.2"]
+                 [binaryage/devtools "0.9.4"]
                  [figwheel-sidecar "0.5.9" :exclusions [ring/ring-core joda-time org.clojure/tools.reader]]
                  [com.cemerick/piggieback "0.2.1"]
                  [org.clojure/tools.nrepl "0.2.12"]
+                 [org.clojure/core.async "0.3.442"]
+                 [org.clojure/tools.reader "1.0.0-beta4"]
                  [juxt/dirwatch "0.2.3"]
                  [garden "1.3.2"]
                  [untangled/om-css "1.0.0"]
                  [com.rpl/specter "0.13.0"]
                  [navis/untangled-client "0.8.2-SNAPSHOT" :scope "provided"]
                  [navis/untangled-server "0.7.0-SNAPSHOT" :scope "provided"]
                  [navis/untangled-spec "0.3.7-1"]
-                 [navis/untangled-datomic "0.4.9" :exclusions [com.datomic/datomic-free org.clojure/tools.cli]]
-                 [navis/untangled-ui "0.1.0-SNAPSHOT"]
-                 ]
+                 [navis/untangled-ui "0.1.0-SNAPSHOT"]]
 
-  :plugins [[lein-cljsbuild "1.1.3"]]
+  :plugins [[lein-cljsbuild "1.1.5"]]
```

Comment:
This is a biggie, and was probably a problem. Latest cljsbuild is always a good idea. I've seen all sorts of badness from being behind on this.